src/simple.rs

```diff
@@ -17,6 +18,31 @@ pub fn visible_block_faces<T, S>(
 ) where
     T: Voxel,
     S: Shape<u32, 3>,
+{
+    visible_block_faces_with_cast::<_, IdentityVoxel<T>, _>(
```

Comment:
I'm not sure "cast" is the proper term for this. Maybe `_with_voxel_view` or something.

Reply:
Done.